tests/aws/services/events/test_events.py

```diff
@@ -76,6 +81,81 @@
 
 
 class TestEvents:
+    @markers.aws.validated
+    @pytest.mark.skipif(
+        not is_v2_provider(),
+        reason="V1 provider does not support this feature",
+    )
+    def test_put_events_without_source(self, snapshot, aws_client):
+        entries = [
+            {
+                "DetailType": TEST_EVENT_PATTERN_NO_SOURCE["detail-type"][0],
+                "Detail": json.dumps(EVENT_DETAIL),
+            },
+        ]
+        response = aws_client.events.put_events(Entries=entries)
+        snapshot.match("put-events", response)
+
+    @markers.aws.unknown
```

Comment:
Is this test not yet aws-validated?

Reply:
fixed in: https://github.com/localstack/localstack/pull/10664/commits/d2b4586aebd3486090d03bc97244f06873ed1eed